src/qt/intro.cpp

```diff
@@ -322,6 +305,9 @@ void Intro::startThread()
 
     connect(executor, &FreespaceChecker::reply, this, &Intro::setStatus);
     connect(this, &Intro::requestCheck, executor, &FreespaceChecker::check);
+    connect(ui->prune, &QCheckBox::stateChanged, [this](int state) {
+        setSizeWarningLabel(state == 2);
```

Comment:
Could get rid of the magic number `2`?
See:
- https://doc.qt.io/qt-5/qt.html#CheckState-enum
- https://doc.qt.io/qt-5/qabstractbutton.html#checked-prop

Reply:
Sure, I will work on your suggested changes now

Reply:
ea8ef9cde7b28fda72aa2313c92e6396e700e357